autorole/assets/autorole.html

```diff
@@ -42,6 +42,12 @@ <h2>Autorole</h2>
                             </select>
                         </div>
                         <div class="form-group">
+                            <h4>Notice:</h4>
+                            <p>
+                            Automatically adding any role on join bypasses Discord's protections such as
+                            <a href="https://support.discord.com/hc/en-us/articles/216679607-What-are-Verification-levels-"><b>Verification Levels</b></a>
```

Comment:
add target="_blank" to the anchor tags to make sure they open in a new window.

Reply:
Done in 1b54652, thanks for the review!